bundle/src/main/java/com/adobe/acs/commons/replication/status/impl/JcrPackageReplicationStatusEventHandler.java

```diff
@@ -459,13 +491,34 @@ protected void activate(final Map<String, String> config) throws LoginException
             this.replicatedAt = ReplicatedAt.PACKAGE_LAST_MODIFIED;
         }
 
-        this.replicationStatusNodeTypes = PropertiesUtil.toStringArray(config.get(PROP_REPLICATION_STATUS_NODE_TYPES),
+        final String[] nodeTypeAndPathRestrictions = PropertiesUtil.toStringArray(config.get(PROP_REPLICATION_STATUS_NODE_TYPES),
                 DEFAULT_REPLICATION_STATUS_NODE_TYPES);
 
+        // the map must keep the order!
+        pathRestrictionByNodeType = new LinkedHashMap<>();
+        for (String nodeTypeAndPathRestrictionEntry : nodeTypeAndPathRestrictions) {
+            Map.Entry<String, Pattern> nodeTypeAndPathRestriction = extractNodeTypeRestrictionAndPathRestrictionFromConfigEntry(nodeTypeAndPathRestrictionEntry);
+            pathRestrictionByNodeType.put(nodeTypeAndPathRestriction.getKey(), nodeTypeAndPathRestriction.getValue());
+        }
         log.info("Package Replication Status - Replicated By Override User: [ {} ]", this.replicatedByOverride);
         log.info("Package Replication Status - Replicated At: [ {} ]", this.replicatedAt.toString());
-        log.info("Package Replication Status - Node Types: [ {} ]",
-                StringUtils.join(this.replicationStatusNodeTypes, ", "));
+        log.info("Package Replication Status - Node Types and Path Restrictions: [ {} ]", pathRestrictionByNodeType);
+    }
+
+    protected Map.Entry<String, Pattern> extractNodeTypeRestrictionAndPathRestrictionFromConfigEntry(String entry) {
+        // split up between node types and paths
+        int separatorPos = entry.indexOf(' ');
```

Comment:
More just an FYI as this doesn't quite fit your use case.. but we have ParameterUtil which helps w stuff like this (again, don't think it quite fits your use case without modification).

https://github.com/Adobe-Consulting-Services/acs-aem-commons/blob/665faaa8beff5f314f0d384ff4d2abbc820f1691/bundle/src/main/java/com/adobe/acs/commons/util/ParameterUtil.java